examples/hardhat-hts-crypto-transfer-hbar/contracts/IHederaTokenService.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
we need to export this interface (and others like IERC20/IERC721) from `contracts/`.

Opened a ticket here https://github.com/hashgraph/hedera-forking/issues/227